doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md

```diff
@@ -0,0 +1,332 @@
+#      in either TransportSystems/VacuumBoundary or TransportSystems/ReflectingBoundary
+#      but not both
+
+[Mesh]
+  [eqv_core]
+    type = EquivalentCoreMeshGenerator
+    input = het_core
+    target_geometry = full_hom
+  []
+  [block_numbering]
+    type = ElementIDToSubdomainID
+    input = 'eqv_core'
+    extra_element_id_name = 'material_id'
+  []
+  data_driven_generator = eqv_core
+[]
+
+[GlobalParams]
+  is_meter     = false
+  plus         = true
+  library_file = 'mcc3xs.xml'
+  library_name = 'ISOTXS-neutron'
+[]
+
+[TransportSystems]
+  particle = neutron
+  equation_type = eigenvalue
+  G = 33
+  ReflectingBoundary = 'outer_core top bottom'
+  VacuumBoundary = 'outer_core top bottom'
+  [dfem_sn]
+    scheme = DFEM-SN
+    family = L2_LAGRANGE
+    order = FIRST
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 1
+    using_array_variable = true
+    using_averaged_xs = true
+    direction_major_ordering = true
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+  richardson_rel_tol = 1e-6
+  richardson_abs_tol = 1e-6
+  richardson_max_its = 100
+  richardson_value = eigenvalue
+  inner_solve_type = GMRes
+  max_inner_its = 4
+  inner_tol = 1.e-3
+  cmfd_acceleration = true
+[]
+
+[Materials]
+  [material_core_grid1]
+    type = MixedMatIDNeutronicsMaterial
+    isotopes = 'pseudo_REG'
+    densities = '1.0'
+    block = '100 300 400 700 800 900 1000'
+    grid_names = 'Tcool'
+    grid = '1'
+  []
+  [material_core_grid2]
+    type = MixedMatIDNeutronicsMaterial
+    isotopes = 'pseudo_REG'
+    densities = '1.0'
+    block = '200 500 600'
+    grid_names = 'Tcool Tfuel'
+    grid = '1 1'
+  []
+[]
+```
+
+Users are encouraged to look through this input file in detail and update input parameters accordingly based on their modeling needs. At a minimum however, a mesh file related to the heterogeneous reactor core needs to be defined with a name that matches `Mesh/eqv_core/input`, and boundary conditions for the full-core eigenvalue problem need to be defined by setting the appropriate outer boundary sidesets using `TransportSystems/VacuumBoundary` and `TransportSystems/ReflectingBoundary`. By looking at this file, it can be seen that the mapping between mesh blocks and cross section material ID is automatically done by calling `MixedMatIDNeutronicsMaterial` in the `[Materials]` block, and this assumes that the homogeneous input reactor core mesh has been generated with `EquivalentCoreMeshGenerator`.
+
+The heterogeneous reactor core mesh can be copied directly from the `xs_generation_mesh.i` (everything up to and including the `Mesh/het_core` block), and pasted into `core_hom_macro.i` above the `Mesh/eqv_core` block. The remainder of the `[Mesh]` block in `core_hom_macro.i` follows very similar steps as the ones explained in `xs_generation_mesh.i`, `EquivalentCoreMeshGenerator` is responsible for generating the homogeneous mesh from the input heterogeneous mesh specifications.
```

Comment:
I wonder if you could
!include that input file instead, with the right 'active/inactive' parameter in [Mesh]
or maybe with a final_generator_name

Reply:
@GiudGiud in case it wasn't clear through the documentation, Griffin will automatically generate a `core_hom_macro.i` file that will be the starting point for running the full core solve step. The user will have to modify this file slightly to get it to actually run for this problem, so I'm wondering if it makes sense to just commit a file `core_hom_macro_updated.i` that has all the additions needs for the full core solve step to properly run. Previously I was trying to do this all through cli args in order to make sure that the process of file generation and file editing could be tested properly, but the downside of this is that the documentation page does not have a file `core_hom_macro.i` to reference. What do you suggest?

Reply:
The other way this could potentially be simplified is to define another file `abtr_het_mesh.i` which contains the shared mesh blocks between `xs_generation_mesh.i` and `core_hom_macro.i`, since the het mesh generation step is exactly the same in both cases.

Reply:
Basically, the slightly confusing thing here is explaining what changes are needed to `core_hom_macro.i` to define the Mesh block. Mesh blocks needed to be added to both the beginning (het mesh definition) and the end of the mesh block (coarse mesh for equivalent core) in `core_hom_macro.i`

Reply:
@GiudGiud I've gone ahead and added the file `abtr_het_mesh.i`, which has all the meshing routines that are common to both xs generation and full core solve steps. The only block in the documentation that is manually defined is the `[coarse_mesh]` block, since this is the only block that is specific to the full core solve step. Let me know if this is sufficient